files/install-deps-worker.yml

```diff
@@ -12,16 +12,28 @@
           - python3-click
           - python3-GitPython
           - python3-sh
-          - rebase-helper
           - python3-timeout-decorator
           - python3-celery
           - python3-redis
           - python3-lazy-object-proxy
           - redis # redis-cli for debugging
           - python3-cccolutils # packitos dependency, otherwise we'd need gcc to compile it
+          - gcc
+          - git
+          - krb5-devel
+          - python3-devel
+          - python3-pip
+          - rpm-build
     - name: Install pip deps
       pip:
         name:
-          - sentry-sdk
+          # Using GitHub because
+          # a) rebasehelper has some problem being installed from PyPI
+          #    (event it works from inside a container)
+          # b) We can freeze the source-code version
+          #    and choose any git ref of the packages.
+          - git+https://github.com/getsentry/sentry-python@0.19.1
+          - git+https://github.com/rebase-helper/rebase-helper@0.23.1
```

Comment:
I don't mind this solution, just FYI that
```yaml
          - sentry-sdk==0.19.1
          - rebasehelper==0.23.1
```
works OK for me

Reply:
I know the syntax, but wanted to use `rebasehelper` first and it did not build. I'll try with the version and see if it helps.

Reply:
Ok, it works.